museek/data_element.py

```diff
@@ -121,6 +135,13 @@ def _std(self, axis: int | list[int, int] | tuple[int, int]) -> 'DataElement':
         """ Return a `DataElement` created from the output of `np.std` applied along `axis`. """
         return DataElement(array=np.std(self.array, axis=axis, keepdims=True))
 
+    def _kurtosis(self):
```

Comment:
we should test `kurtosis()`, `_kurtosis()` and `flagged_kurtosis()` separately

Reply:
(at least 1 test for each)

Reply:
I think you have `kurtosis()` and `_kurtosis()` already, but `_flagged_kurtosis()` is missing, right?